specification/README.md

```diff
@@ -0,0 +1,21 @@
+# Semantic Conventions
```

Comment:
Given [opentelemetry.io#2908](https://github.com/open-telemetry/opentelemetry.io/pull/2908) can you make the title be exactly `# OpenTelemetry Semantic Conventions`?